lib/src/searchfield.dart

```diff
@@ -342,7 +351,7 @@ class _SearchFieldState extends State<SearchField> {
             stream: sourceStream.stream,
             builder:
                 (BuildContext context, AsyncSnapshot<List<String?>?> snapshot) {
-              late int count = widget.maxSuggestionsInViewPort;
+              late var count = widget.maxSuggestionsInViewPort;
```

Comment:
Any specific reason why we should user `var` over `int`. AFAIK we should always specify types if as long as we know it ahead of compile time.

Reply:
Looking at this reference https://dart-lang.github.io/linter/lints/omit_local_variable_types.html, the dart linter suggests omitting local variable types, since they can easily be inferred. But you are right, we should specify types ahead of compile time, but since it is a local variable, this does not seem to be necessary.